app/src/main/res/resource/values-de/strings.xml

```diff
@@ -26,17 +26,17 @@
     <string name="search">SUCHE</string>
     <string name="follow">FOLGEN</string>
     <string name="following">FOLGT</string>
-    <string name="my_follow">Meine Folgungen</string>
+    <string name="my_follow">Anhänger</string>
```

Comment:
Oh no, please not "Anhänger". These are maybe religious or fans of a band, but in social media you just use "Follower" as in English. I've never heard a better word. Also don't forget the "my".
--> "Meine Follower"